modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/DataSetFilterCETImpl.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.client.extension.type.internal;
+
+import com.liferay.client.extension.constants.ClientExtensionEntryConstants;
+import com.liferay.client.extension.model.ClientExtensionEntry;
+import com.liferay.client.extension.type.DataSetFilterCET;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.kernel.util.UnicodeProperties;
+import com.liferay.portal.kernel.util.UnicodePropertiesBuilder;
+
+import java.util.Properties;
+import java.util.Set;
+
+import javax.portlet.PortletRequest;
+
+/**
+ * @author Bryce Osterhaus
+ */
+public class DataSetFilterCETImpl
+	extends BaseCETImpl implements DataSetFilterCET {
+
+	public DataSetFilterCETImpl(ClientExtensionEntry clientExtensionEntry) {
+		super(clientExtensionEntry);
+	}
+
+	public DataSetFilterCETImpl(PortletRequest portletRequest) {
+		this(
+			StringPool.BLANK,
+			UnicodePropertiesBuilder.create(
+				true
+			).put(
+				"url", ParamUtil.getString(portletRequest, "url")
+			).build());
+	}
+
+	public DataSetFilterCETImpl(
+		String baseURL, long companyId, String description,
+		String externalReferenceCode, String name, Properties properties,
+		String sourceCodeURL, UnicodeProperties typeSettingsUnicodeProperties) {
+
+		super(
+			baseURL, companyId, description, externalReferenceCode, name,
+			properties, sourceCodeURL, typeSettingsUnicodeProperties);
+	}
+
+	public DataSetFilterCETImpl(
+		String baseURL, UnicodeProperties typeSettingsUnicodeProperties) {
+
+		super(baseURL, typeSettingsUnicodeProperties);
+	}
+
+	@Override
+	public String getEditJSP() {
+		return "/admin/edit_data_set_filter.jsp";
+	}
+
+	@Override
+	public String getType() {
+		return ClientExtensionEntryConstants.TYPE_DATA_SET_FILTER;
```

Comment:
A nit about naming, as we're creating a filter for the FDS, we shall use "TYPE_FDS_FILTER" instead, in line with how we already named the cell renderer